### PR TITLE
Fix - The modal name "Confirm Operation" is one for all operations

### PR DESCRIPTION
**Pull request by LendiHop**

### CODE REVIEW

File: src/modals/confirmation-modal/d-app-operations-confirmation/permission-request-confirmation/permission-request-confirmation.tsx [hunk 1]
```diff
@@ -72,6 +74,8 @@ export const PermissionRequestConfirmation: FC<Props> = ({ message }) => {
       publicKey: approver.publicKey
     });
 
+  useNavigationSetOptions({ headerTitle: () => <HeaderTitle title={'Confirm Connection'} /> }, []);
```
Comment (unixvb) [inline src/modals/confirmation-modal/d-app-operations-confirmation/permission-request-confirmation/permission-request-confirmation.tsx:77]: according to style guide if you are using hardcoded string as prop do not use curly braces
`title="Confirm Connection"`

File: src/modals/confirmation-modal/d-app-operations-confirmation/sign-payload-request-confirmation/sign-payload-request-confirmation.tsx [hunk 2]
```diff
@@ -63,6 +65,8 @@ export const SignPayloadRequestConfirmation: FC<Props> = ({ message }) => {
     [accounts, message.sourceAddress]
   );
 
+  useNavigationSetOptions({ headerTitle: () => <HeaderTitle title={'Confirm Sign'} /> }, []);
```
Comment (unixvb) [inline src/modals/confirmation-modal/d-app-operations-confirmation/sign-payload-request-confirmation/sign-payload-request-confirmation.tsx:68]: the same here

File: src/modals/confirmation-modal/internal-operations-confirmation/internal-operations-confirmation.tsx [hunk 3]
```diff
@@ -13,6 +15,26 @@ export const InternalOperationsConfirmation: FC<Props> = ({ opParams }) => {
...
+            title = 'Confirm Send';
+            break;
+        }
+
+        return <HeaderTitle title={title} />;
+      }
+    },
+    []
+  );
+
```
Comment (unixvb) [inline src/modals/confirmation-modal/internal-operations-confirmation/internal-operations-confirmation.tsx:37]: what would happen if we will call neither `delegation` nor `transaction`?

**Comment by unixvb** [comment]
Make PR title human readable

File: src/modals/confirmation-modal/internal-operations-confirmation/internal-operations-confirmation.tsx [hunk 4]
```diff
@@ -13,6 +15,26 @@ export const InternalOperationsConfirmation: FC<Props> = ({ opParams }) => {
 
   const selectedAccount = useSelectedAccountSelector();
 
+  let title = 'Confirm Operation';
```
Comment (unixvb) [inline src/modals/confirmation-modal/internal-operations-confirmation/internal-operations-confirmation.tsx:18]: no need of this variable in the component scope. move it inside anonymous fn

**Review by unixvb — APPROVED** [review]

**Review by andrewkryshtal — APPROVED** [review]

**Pull request merged**